components/partials/AlgoliaSearchInput.vue

```diff
@@ -1,7 +1,12 @@
 <template>
   <div
     class="algolia-wrapper relative"
-    :class="[focused ? 'focused' : 'blurred', !q && 'blurred', appearance]"
+    :class="[
```

Comment:
If `html[dir='ltr']` is used to target the CSS class, no need for dynamic class to handle language direction